.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # composer
 /vendor/
+composer.lock
```

Comment:
I think there is a benefit in committing the `composer.lock`

Reply:
done :)